.travis.yml

```diff
@@ -9,13 +9,10 @@ before_install:
 install:
   - npm i -g npm
   - npm install
-  - rvm install 2.3.1
-  - rvm use 2.3.1
-  - gem install danger --version '~> 3.0'
 before_script:
   - ruby -v
```

Comment:
If we're no longer installing Ruby, do we need this line? I noticed there's a gemrc file being output as part of the build but I don't know very much about Ruby. Is there anything still using Ruby in the project?